### PR TITLE
Subject: Core structure of app and vehicle control

### LINKED ISSUE #3: As a security manager, I would like to choose the route of the surveillance vehicle so that I can surveil the area that I am responsible for.
**Acceptance Criteria**
The mobile device should have two buttons that the user can press:
- Circle-button
- Square-button

Pressing the button should display a message "Pressed a button".

### LINKED ISSUE #12: Find bluetooth devices in app
**Acceptance criteria:**
- Display list of available bluetooth devices
- Turn on bluetooth on mobile phone
- Turn off bluetooth on mobile phone
- Display list of paired devices
- Display successful connection message

### LINKED ISSUE #14: Serial reader mobile application
Create a mobile application that can receive that can receive serial prints from the car.

**Acceptance criteria:**
- Receive a message from the Arduino and displaying it on the smartphone app

### PR DESCRIPTION
**Pull request by KarlWestgardh**
## Description
Basic layout of app
App structure, fragments, activity, interface and XML
Bluetooth connectivity
Templates for commit message, pull request, issue template
Basic movement of vehicle

## Related issues
Fixes #12 
Fixes #14 
Fixes #3 
Relates to #24 
Relates to #23 
Relates to #22 
Relates to #11

### CODE REVIEW

**Comment by platisd** [comment]
Did you get bluetooth connectivity already? Nice!

**Comment by EmanuelDellsen** [comment]
A lot of hours and googling resulted in connecting and getting input from
the car but writing to it and parsing to the car seems trickier.. But the
struggle continues!
 Have a good weekend,
Best regards,
Emanuel

File: arduino/src/hajkenCar/hajkenCar.ino [hunk 1]
```diff
@@ -0,0 +1,150 @@
...
+void obstacleAvoidance() {
+  if(obstacleAvoidanceOn)
+  {car.update(); 
+  carDistanceToObstacle = USSensorFront.ping_cm(); // UltraSonicSound Sensor measures (0 = more than 100 cm distance)
+  if (carDistanceToObstacle <= stopDistanceToObstacle && carDistanceToObstacle > 0) {
+    Serial2.write("Obstacle detected"); // Sending message to bluetooth
+    car.setSpeed(stopSpeed);
+    car.update();
+  }}
+}
```
Comment (platisd) [inline arduino/src/hajkenCar/hajkenCar.ino]: Just a tip: If you press "CTRL+t" your code will be indented nicely by the Arduino IDE.

Comment (strand93) [reply arduino/src/hajkenCar/hajkenCar.ino]: Thanks! Nice to know .

**Pull request merged**